frontend/src/components/Meeting.js

```diff
@@ -0,0 +1,54 @@
+import React, { forwardRef, useState } from "react";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Checkbox from "@material-ui/core/Checkbox";
+import FormGroup from "@material-ui/core/FormGroup";
+import FormControl from "@material-ui/core/FormControl";
+
+const Meeting = forwardRef(({ meeting }, ref) => {
+  const [checked, setChecked] = useState(false);
+  const handleChange = (e) => {
+    setChecked(e.target.checked);
+  };
+  return (
+    <div ref={ref}>
+      <h2 id="meeting-title">Meeting</h2>
+      <h3 id="transition-modal-title">When is the meeting?</h3>
+      <p id="transition-modal-description">July 20, 2014</p>
+      <h3>What is the meeting level?</h3>
+      <p>Junior</p>
+      <h3>What are the topics?</h3>
+      <p>Arithmetic</p>
+      <h3>How can I join?</h3>
+      <p>
+        Join us on <a href="#">Zoom</a> and <a href="#">Miro</a>
```

Comment:
Im sure the warnings here will be fixed down the road.

Reply:
Yeah, many variables will be used when there are actual data